options.go

```diff
@@ -40,6 +40,7 @@ type options struct {
 	FcBinary           string   `long:"firecracker-binary" description:"Path to firecracker binary"`
 	FcKernelImage      string   `long:"kernel" description:"Path to the kernel image" default:"./vmlinux"`
 	FcKernelCmdLine    string   `long:"kernel-opts" description:"Kernel commandline" default:"ro console=ttyS0 noapic reboot=k panic=1 pci=off nomodules"`
+	FcInitrdPath       string   `long:"initrd-path" description:"path to initrd" default:"./initramfs"`
```

Comment:
3 comments:
1) Since this is optional, I don't think we should have a default.
2) We don't suffix other options with `path`, I think the field name and arg name should just be `FcInitrd` and `initrd` respectively. 
3) Nitpick: capitalize the first letter of the description (i.e. "Path")

Reply:
Thanks for the feedback, pushed up a fix for all 3 points.